Linux Script/configure-linux.sh

```diff
@@ -350,10 +353,15 @@ checkIfLogglyServersAccessible() {
   fi
 
   echo "INFO: Checking if '$LOGGLY_ACCOUNT' subdomain is valid."
-  if [ $(curl --head -s --request GET $LOGGLY_ACCOUNT_URL/login | grep "200 OK\|HTTP/2 200" | wc -l) ] >0; then
-    echo "INFO: $LOGGLY_ACCOUNT_URL is valid and reachable."
+  if [[ $LOGGLY_ACCOUNT != $INVALID_SUBDOMAIN ]]; then
+      if [[ $(curl --head -s --request GET $LOGGLY_ACCOUNT_URL/login | grep "200 OK\|HTTP/2 200") ]]; then
```

Comment:
I modified this condition to grep only the **200 response code** and then mark the subdomain valid.